make-release.sh

```diff
@@ -7,12 +7,28 @@ SCRIPT_DIR=$(cd "$(dirname "$0")" || exit; pwd)
 REGISTRY="quay.io"
 ORGANIZATION="eclipse"
 
-function die_with() 
+die_with() 
 {
 	echo "$*" >&2
 	exit 1
 }
 
+
+usage ()
+{
+  echo "Usage: $0  --version [CHE VERSION TO RELEASE] --dwo-version [DEVWORKSPACE OPERATOR VERSION TO RELEASE] --phases [LIST OF PHASES]
+
+Phases are comma-separated list, e.g. '1,2,3,4,5,6', where each phase has its associated projects:
+#1: MachineExec, CheTheia, DevfileRegistry, Dashboard, DwoOperator, JWTProxyAndKIP; 
+#2: CheServer; 
+#3: CheTheia; 
+#4: ChePluginRegistry
+#5: DwoCheOperator; 
+#6: CheOperator; "
+
+  echo "Example: $0 --version 7.29.0 --dwo-version 0.3.0 --phases 1,2,3,4,5,6"; echo
+}
```

Comment:
since any time we echo the usage, we also exit, probably makes sense to add an `exit 1` here, unless you plan to add a `-h` flag that returns `exit 0` to differentiate between a failure and a request for help (success).